examples/asr/transcribe_speech.py

```diff
@@ -151,10 +232,22 @@ def main(cfg: TranscriptionConfig) -> TranscriptionConfig:
     else:
         # get filenames from manifest
         filepaths = []
+        if os.stat(cfg.dataset_manifest).st_size == 0:
+            logging.error(f"The input dataset_manifest {cfg.dataset_manifest} is empty. Exiting!")
+            return None
+
         with open(cfg.dataset_manifest, 'r') as f:
+            partial_audio = True
             for line in f:
                 item = json.loads(line)
+                if "duration" not in item:
```

Comment:
duration is not the key factor determining limit on audio - it is both duration + offset presence that determines that partial audio is chosen. 

Duration is filled in for nearly all manifests always.